### PR TITLE
fix(Grid): Update grid layout to support new 2xl breakpoint

### LINKED ISSUE #2199: Update grid layout to support new 2xl breakpoint
Update the grid layout to support the 2xl breakpoint added here - https://github.com/patternfly/patternfly-next/pull/1880

### PR DESCRIPTION
**Pull request by jessiehuff**
Fixes #2199

Update the grid layout to support the 2xl breakpoint added here - patternfly/patternfly-next#1880

### CODE REVIEW

File: packages/patternfly-4/react-core/src/layouts/Grid/Grid.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ const propTypes = {
   lg: gridItemSpanValueShape,
   /** the number of columns all grid items should span on a xLarge device */
   xl: gridItemSpanValueShape,
+  /** the number of columns all grid items should span on a 2xLarge device */
+  xl2: gridItemSpanValueShape,
```
Comment (mcoker) [inline packages/patternfly-4/react-core/src/layouts/Grid/Grid.js]: should we call it `2xl` or is there a reason it's `xl2` here?

Comment (jessiehuff) [reply packages/patternfly-4/react-core/src/layouts/Grid/Grid.js]: I had to name it xl2 to meet JSX attribute name requirements. It won't allow a prop to start with a number. :(

Comment (boaz0) [reply packages/patternfly-4/react-core/src/layouts/Grid/Grid.js]: Maybe `'2xl': gridItemSpanValueShape` will make JSX happy?

Comment (jessiehuff) [reply packages/patternfly-4/react-core/src/layouts/Grid/Grid.js]: That works for defining the prop, but it doesn't work when actually creating a `<GridItem 2xl={2}>`

**Pull request merged**